sources/models/contacts.js

```diff
@@ -1,9 +1,32 @@
+const dateToStr = webix.Date.dateToStr("%Y-%m-%d");
```

Comment:
The server accepts all dates only as "%Y-%m-%d %H:%i"
I understand it may be confusing here, but you need to check if the dates were saved correctly anyway.

Reply:
Fixed.